src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -448,6 +451,16 @@ class ActivityFeed extends React.Component<Props, State> {
 
         feedItems.sort((a, b) => Date.parse(b.created_at) - Date.parse(a.created_at));
 
+        const { commentsError, tasksError, versionError } = this.props;
```

Comment:
Instead of three separate errors, why not just a `feedItemsError` prop?

Are there cases in the future where we want to react differently for each one?

Reply:
I left versions separate since we send that by itself to the details sidebar